migrations/1593197631571_add-closed-time-to-action-items.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+    pgm.addColumn('action_items', {
+        closed_at: { type: 'timestamp' }
```

Comment:
do we want to make a distinction between "completed" and "canceled"?

Reply:
yeah. I was going to have status be OPEN | COMPLETED | CANCELED, where the last two are types of "closed". So when there's a `closed_at` timestamp, there will also be a competed or canceled status

Reply:
also lazily not representing that in another table or anything 🙃